### PR TITLE
Adding NACLAdminitrators group and policies.

### PR DESCRIPTION
**Pull request by RichardWeiss**

### CODE REVIEW

File: vpc/vpc-meta.template [hunk 1]
```diff
@@ -415,4 +442,4 @@
       }
     }
   }
-}
+}
```
Comment (gozer) [inline vpc/vpc-meta.template:445]: Spurious whitespace change

**Comment by gozer** [comment]
Linking to an issue would be appreciated, thx!

**Comment by limed** [comment]
Issue #203

**Comment by tinnightcap** [comment]
r+

**Pull request merged**